src/bin/busd.rs

```diff
@@ -86,7 +86,7 @@ async fn main() -> Result<()> {
     }
 
     if args.print_address {
-        println!("{}", bus.address());
+        info!("{}", bus.address());
```

Comment:
This should still use `println` as user is explicitly asking for printing the address to stdout (which they may pipe to a file or another app).